background/nudge/activities.go

```diff
@@ -154,3 +134,28 @@ func (n *NudgeWorker) NotifySymptomFollowUpActivity(ctx context.Context, account
 
 	return n.mongo.UpdateAccountSymptomNudge(accountNumber)
 }
+
+// NotifySymptomSpikeActivity send notifications to accounts those have symptoms spiked around
+func (n *NudgeWorker) NotifySymptomSpikeActivity(ctx context.Context, accountNumber string, symptoms []schema.Symptom) error {
+	logger := activity.GetLogger(ctx)
+
+	logger.Info("Prepare the message context for following up symptoms", zap.Any("symptoms", symptoms))
+
+	var symptomsIDs = make([]schema.SymptomType, 0)
+	for _, s := range symptoms {
+		symptomsIDs = append(symptomsIDs, s.ID)
+	}
+
+	headings, contents, err := SymptomListingMessage("symptom_spike", symptoms)
+	if err != nil {
```

Comment:
And how about this error? It's not returned?

Reply:
Will fix.